keras/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "keras" %}
+{% set version = "1.2.2" %}
+{% set sha256 = "786f7a98a09c1217e7c76c2fb485440c3675c7188bf7d9afcd4e5c75fc97406e" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/fchollet/keras/archive/{{ version }}.tar.gz
+  fn: keras-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - h5py
+    - numpy
+    - setuptools
+    - scipy
+    - tensorflow <=1.0  # [not win]
```

Comment:
wait what

Reply:
Only tensorflow versions up to 1.0 are support for this version of keras and there were only builds for linux and osx back then. Windows builds were supported later (but possibly only on python 3.6).

Reply:
> Only tensorflow versions up to 1.0 are support for this version of keras

Do we have code somewhere which uses modern keras + tensorflow? I thought we did but maybe my memory is bad.

Reply:
Keras.jl works on the latest version, but we aren't using that package internally.